src/Workspaces/Remote/ServiceHub/Services/DiagnosticAnalyzer/DiagnosticComputer.cs

```diff
@@ -503,6 +504,14 @@ private static (ImmutableArray<DiagnosticAnalyzer> projectAnalyzers, ImmutableAr
             if (analyzerMap.TryGetValue(analyzerId, out var analyzer))
             {
                 hostBuilder.Add(analyzer);
+
+                if (analyzer is DiagnosticSuppressor)
+                {
+                    if (hostBuilderSuppressorRange.Equals(default))
+                        hostBuilderSuppressorRange = new Range(hostBuilder.Count - 1, hostBuilder.Count);
+                    else
+                        hostBuilderSuppressorRange = new Range(hostBuilderSuppressorRange.Start, hostBuilder.Count);
```

Comment:
can we just grab out the suppressors we care about?  all this work to have ranges seems unnecessary.

Reply:
➡️ Switched to different approach